index.html

```diff
@@ -52,13 +53,12 @@
         </button>
       </div>
     </section>
-    <section class="container mx-12 mt-12 mb-12">
-      <div class="grid gap-8 md:grid-cols-2 lg:grid-cols-4">
+    <section class="  md:mx-12 mx-12  md:mb-12 mb-8">
```

Comment:
Code formatting issues...
```suggestion
    <section class="md:mx-12 mx-12 md:mb-12 mb-8">
```